packages/admin-next/dashboard/src/components/table/data-table/data-table-root/data-table-root.tsx

```diff
@@ -52,6 +52,10 @@ export interface DataTableRootProps<TData> {
    * Whether the table is empty due to no results from the active query
    */
   noResults?: boolean
+  /**
+   * Whether to display the tables header
```

Comment:
TBH I'm not a big fan of "negative" naming for props, but wanted to stay consistent with other props in this component